pyapp/cookiecutter.json

```diff
@@ -3,7 +3,7 @@
     "package_name": "{{ cookiecutter.name.lower().replace(' ', '_').replace('-', '_') }}",
     "slug": "{{ cookiecutter.package_name.replace('_', '-') }}",
     "short_description": "A Python app.",
-    "python_version": "3.12.0",
+    "python_version": "3.11.7",
```

Comment:
The `pyapp` cookiecutter was using `3.12.0` for new apps for some reason. Let's change it to `3.11.7` to match the `pyramid-app` cookiecutter